docs/hazmat/primitives/aead.rst

```diff
@@ -0,0 +1,71 @@
+.. hazmat::
+
+
+Authenticated encryption
+========================
+
+.. module:: cryptography.hazmat.primitives.ciphers.aead
+
+Authenticated encryption with additional data allows the user to guarantee
```

Comment:
I'd dumb the description down "Authenticated encryption with additional data are encryption schemes which provide both confidentiality and integrity for their ciphertext, and also support providing integrity for additional data which is not encrypted"